### DISCUSSION TITLE
DisplayPromptAsync and landscape mode

### DISCUSSION THREAD

**Discussion by sk479** [dotnet/maui#23226] [Upgrading from Xamarin]
We have a formerly Xamarin and now MAUI application that requires user input via the soft keyboard.
When the application is running in landscape mode on Android, the user input prompt is hidden by the soft keyboard which requires the user to enter their desired text, press Done on the keyboard, then press OK at the display prompt.
On iOS this does not occur and the keyboard appears under the display prompt window as expected.
What can be done to prevent the soft keyboard from hiding the display prompt window?

**Comment by prabhav-mehra** [comment]
Have you tried

https://learn.microsoft.com/en-us/dotnet/maui/android/platform-specifics/soft-keyboard-input-mode?view=net-maui-8.0

```xml
<Application ...
             xmlns:android="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;assembly=Microsoft.Maui.Controls"
             android:Application.WindowSoftInputModeAdjust="Resize">
  ...
</Application>
```

**Comment by sk479** [comment]
Thanks for your reply.

Yes we have tried using the keyboard resize option listed in the article.

It produced no different results.
